site/docs/get-started/index.md

```diff
@@ -37,6 +37,7 @@ platform. We describe three easy options:
  - [Vagrant](/docs/get-started/vagrant) -- run ManageIQ as a Vagrant Box
  - [Docker](/docs/get-started/docker) -- run ManageIQ as a Docker container
  - [Public Cloud](/docs/get-started/cloud) -- run ManageIQ in the cloud
+ - [Kubernetes](/docs/get-started/kubernetes) -- run ManageIQ in Kubernetes
```

Comment:
If we're adding Kubernetes here, do we want to get rid of the paragraph above about OpenShift/Kubernetes?

Reply:
I'm not sure. I feel like the other options in this list are specifically not production-ready options. The pods repo is meant for production, but is also not overly difficult to deploy in a few steps.

Reply:
Oh I see..  Then, maybe "however" should be removed from "For this getting started guide however"?

Reply:
Sure, I'm not sure how important the exact language here is. I'll update it a bit.

Reply:
It just reads strange as is, so I mentioned. But it's obviously not important enough to keep exchanging comments, so I'll stop now 😄

Reply:
Yeah, I agree, but I'm not really sure where else we would go with it.